setup.py

```diff
@@ -76,7 +76,7 @@ def run(self):
         'tornado>=4.1',
         'nest-asyncio>=1.5',
     ],
-    python_requires='>=3.5',
+    python_requires='>=3.6.1',
```

Comment:
So anyone using 3.5 with 6.1.13 will still have issues. What's the plan there? Does 6.1.13 get revoked/blacklisted somehow and then _this_ change goes into a new 6.2 release?

Reply:
Yes, v6.1.13 has been marked as broken on conda-forge, and I'm waiting for someone with management rights to yank it on PyPI.
When we merge this PR, I will release v6.2.

Reply:
@davidbrochart I can yank 6.1.13. Do you want me to wait for the 6.2 release before yanking? Usually you want a roll-forward option before removing a dep, especially when it's not a security issue.

Reply:
@davidbrochart 6.1.13 is yanked. You should have permission on PyPI for this as well (let me know if you don't see the option)

Reply:
Thanks Matthew. Unfortunately I still don't see admin permission on PyPI.

Reply:
Aww I think I see the issue. Should be fixed now.

Reply:
Yes it is, thanks!